sound/soc/intel/common/soc-acpi-intel-mtl-match.c

```diff
@@ -794,6 +802,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_mtl_sdw_machines[] = {
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-mtl-rt711-l0-rt1316-l3.tplg",
 	},
+	{
+		.link_mask = BIT(0),
+		.links = mtl_cs42l43_l0,
+		.drv_name = "sof_sdw",
+		.sof_tplg_filename = "sof-mtl-cs42l43-l0.tplg",
+	},
```

Comment:
why not insert this after line 791, that way all platforms with cs42l43 will be listed one after the other?

Reply:
Yeah probably good to group them but otherwise this patch looks good to me.

Reply:
updated